### PR TITLE
Update the style of spinner

### PR DESCRIPTION
**Pull request by zdlm**
The back color (the color of the complete circle) has to be @ts-color-gray-light with 50% opacity. The opacity ensures it looks ok against both white backgrounds and an overlay.
The small version needs to be 3px in thickness and the medium needs to be 4px

@esmailpour-hosein @aleks-elkin

### CODE REVIEW

File: packages/core/src/vars.css [hunk 1]
```diff
@@ -1,6 +1,5 @@
 :root {
 	/* Units........................................................ */
-
```
Comment (esmailpour-hosein) [inline packages/core/src/vars.css]: Was this reformat necessary?

File: packages/core/src/vars.css [hunk 2]
```diff
@@ -150,32 +129,27 @@
...
 	--ts-progress-bar-height: 8px;
-
 	--ts-box-width-small: 280px;
 	--ts-box-width-medium: 360px;
-
 	--ts-shadow-color: var(--ts-color-slate-dark);
-
 	/* Spinner........................................................ */
+	--spinner-border-color: rgba(203, 215, 220, 0.5);
```
Comment (esmailpour-hosein) [inline packages/core/src/vars.css]: Can you add a comment to say what's the color name? /* gray-lightest*/ or something like that!

File: packages/core/src/vars.css (hunk 1 above)
Comment (zdlm) [reply packages/core/src/vars.css]: I just run format in my IDE.

File: packages/core/src/vars.css (hunk 2 above)
Comment (zdlm) [reply packages/core/src/vars.css]: 👍

File: packages/core/src/vars.css (hunk 1 above)
Comment (esmailpour-hosein) [reply packages/core/src/vars.css]: Maybe we need to ad more stuff to the .editorconfig file for the future

**Pull request merged**